internal/local/data.go

```diff
@@ -23,15 +23,27 @@ type AppData interface {
 	WriteData(rootDir string) error
 }
 
+// MdbAppMeta represents the .mdb/app_meta.json config
+type MdbAppMeta struct {
+	GroupID       string                 `json:"group_id,omitempty"`
```

Comment:
[q] why are these fields optional? 
I think it makes sense for the `MdbAppMeta` struct to be optional for` local.App` but what reason do we have for making the GroupID/AppID optional? 
maybe this is more a q for @christineschen